extensions/spring-xd-extension-http/src/main/java/org/springframework/integration/x/http/NettyHttpInboundChannelAdapter.java

```diff
@@ -109,6 +114,13 @@
 
 	private final int port;
 
+	/* begin whu */
```

Comment:
We don't need these `begin`, `end` comments - `git blame` provides adequate history as to who changed what code. IDEs such as eclipse even provide a mechanism to show blame in the editor pane.